modules/lti/package.json

```diff
@@ -9,10 +9,14 @@
     "@fortawesome/react-fontawesome": "^0.1.14",
     "axios": "^0.21.1",
     "bootstrap": "^4.6.0",
+    "dangerously-set-html-content": "^1.0.8",
+    "engage-ui": "../engage-ui",
+    "http-mock-server": "^0.5.2",
```

Comment:
This is already listed as devDependency. It's probably not necessary to listr this as runtime dependency as well.

Reply:
These dependencies came from the deeplink PR and are not needed here, removed them.